src/Development/IDE/Spans/Calculate.hs

```diff
@@ -157,40 +175,36 @@ getTypeLPat _ pat =
 
 getLHsType
     :: GhcMonad m
-    => TypecheckedModule
+    => [TypecheckedModule]
     -> LHsType GhcRn
-    -> m [(SpanSource, SrcSpan, Maybe Type)]
-getLHsType _ (L spn (HsTyVar U _ v)) = pure [(Named $ unLoc v, spn, Nothing)]
+    -> m [(SpanSource, SrcSpan, Maybe Type, [T.Text])]
+getLHsType tms (L spn (HsTyVar U _ v)) = do
+  let n = unLoc v
+  docs <- getDocumentationTryGhc' tms n
+  ty <- catchSrcErrors "completion" $ do
+          name' <- lookupName n
+          return $ name' >>= safeTyThingType
+  let ty' = case ty of
+              Right (Just x) -> Just x
+              _ -> Nothing
+  pure [(Named n, spn, ty', docs)]
 getLHsType _ _ = pure []
 
 importInfo :: [(Located ModuleName, Maybe NormalizedFilePath)]
-           -> [(SpanSource, SrcSpan, Maybe Type)]
+           -> [(SpanSource, SrcSpan, Maybe Type, [T.Text])]
```

Comment:
Given that the docs are always empty, I would prefer to add the empty list at the call site instead of returning it from here.